Templates/templates/CSharp/Requests/EntityRequest.cs.tt

```diff
@@ -123,15 +123,17 @@ namespace <#=@namespace#>
                 {
                     <#=thisEntity_Name#>ToInitialize.<#=propertyName#>.AdditionalData = <#=thisEntity_Name#>ToInitialize.AdditionalData;
 
-                    object nextPageLink;
-                    <#=thisEntity_Name#>ToInitialize.AdditionalData.TryGetValue("<#=propertyName.ToLowerFirstChar()#>@odata.nextLink", out nextPageLink);
-                    var nextPageLinkString = nextPageLink as string;
-
-                    if (!string.IsNullOrEmpty(nextPageLinkString))
+                    if(<#=thisEntity_Name#>ToInitialize.AdditionalData.TryGetValue("<#=propertyName.ToLowerFirstChar()#>@odata.nextLink", out var nextPageLink))
```

Comment:
do we still need this one or was it forgotten in the last batch of updates?

Reply:
I had left this out since this was not part of the CollectionResponse but were nested collection properties of an Entity. Let me investigate the viability of doing the same for these as well and get back to you.

Reply:
This should now be resolved via https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/512/commits/6bb3ca74f2d6a844cdcec5183efec060d88b0f5c